2025/using-components-2/demo/2-react/src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import "./styles.css";
+import App from "./App";
+
+createRoot(document.getElementById("root")).render(
```

Comment:
(optional) put the following in both typescript and non-typescript demos so that the code stays consistent without requiring a type assertion: https://github.com/maxpatiiuk/esri-dev-summit-presentations/blob/224034c97f9a02abac37354d493ac008ee2407b8/2025/build-tooling/demo/4-typescript/src/main.tsx#L5-L8